contract-ui/tabs/overview/helpers/getGuidesAndTemplates.ts

```diff
@@ -0,0 +1,386 @@
+  ERC721ClaimPhasesV2: [
+    ALL_GUIDES.realeaseAnNftDropWithNoCode,
+    ALL_GUIDES.biconomyGaslessGuide,
+    ALL_GUIDES.acceptCreditCardPayments,
+    ALL_GUIDES.createGaslessNftDrop,
+    ALL_GUIDES.createNftDropWithClaimPhases,
+  ],
+  ERC721ClaimCustom: [],
+  ERC721LazyMintable: [ALL_GUIDES.createAnMaycCollectionClone],
+  ERC721BatchMintable: [
+    ALL_GUIDES.howToBatchUpload,
+    ALL_GUIDES.releaseAnErc721DropWithCentralizedMetadata,
+  ],
+  ERC721Mintable: [
+    ALL_GUIDES.createNftWithNoCode,
+    ALL_GUIDES.howToRenderNftMetadataInAReactAppUsingThirdwebNftMedia,
+  ],
+  ERC721SignatureMintV2: [
+    ALL_GUIDES.onDemandPass,
+    ALL_GUIDES.mintSpecificNft,
+    ALL_GUIDES.mintNftUniqueCode,
+  ],
+  ERC721SignatureMintV1: [
+    ALL_GUIDES.onDemandPass,
+    ALL_GUIDES.mintSpecificNft,
+    ALL_GUIDES.mintNftUniqueCode,
+  ],
+  ERC721Enumerable: [],
+  ERC721Supply: [
+    ALL_GUIDES.airdropEditionNftsToHoldersOfAnNftCollection,
+    ALL_GUIDES.createAnNftGalleryUsingThirdwebAndNextJs,
+  ],
+  ERC721: [ALL_GUIDES.nftGatedWebsite],
+  ERC1155Burnable: [],
+  ERC1155ClaimConditionsV1: [],
+  ERC1155ClaimConditionsV2: [],
+  ERC1155ClaimPhasesV2: [],
+  ERC1155ClaimPhasesV1: [],
+  ERC1155ClaimCustom: [],
+  ERC1155Revealable: [ALL_GUIDES.nftDropWithDelayedReveal],
+  ERC1155LazyMintableV2: [],
+  ERC1155LazyMintableV1: [],
+  ERC1155SignatureMintable: [ALL_GUIDES.allowGithubContributorsToMintAnNft],
+  ERC1155BatchMintable: [],
+  ERC1155Mintable: [],
+  ERC1155Enumerable: [ALL_GUIDES.createAnNftGalleryUsingThirdwebAndNextJs],
+  ERC1155: [
+    ALL_GUIDES.nftGatedWebsite,
+    ALL_GUIDES.airdropFreeToOwnNftsForAWeb3Game,
+  ],
+  Royalty: [],
+  PrimarySale: [],
+  PlatformFee: [],
+  PermissionsEnumerable: [],
+  Permissions: [],
+  ContractMetadata: [],
+  AppURI: [],
+  Ownable: [],
+};
+
+const ALL_TEMPLATES = {
+  tokenDrop: {
+    title: "Token Drop",
+    url: "https://github.com/thirdweb-example/token-drop",
+  },
+  nftStakingApp: {
+    title: "NFT Staking App",
+    url: "https://github.com/thirdweb-example/nft-staking-app/",
+  },
+  nftDrop: {
+    title: "NFT Drop",
+    url: "https://github.com/thirdweb-example/nft-drop/",
+  },
+  burn1155Mint721: {
+    title: "Burn an ERC1155 to Mint an ERC721 NFT",
+    url: "https://github.com/thirdweb-example/burn1155-mint721",
+  },
+  signatureBasedMinting: {
+    title: "Signature Based Minting",
+    url: "https://github.com/thirdweb-example/signature-based-minting",
+  },
+  nftGatedWebsite: {
+    title: "NFT Gated Website",
+    url: "https://github.com/thirdweb-example/nft-gated-website",
+  },
+  editionDrop: {
+    title: "Edition Drop",
+    url: "https://github.com/thirdweb-example/edition-drop",
+  },
+  packs: {
+    title: "Packs",
+    url: "https://github.com/thirdweb-example/packs",
+  },
+  signatureDrop: {
+    title: "Signature Drop",
+    url: "https://github.com/thirdweb-example/signature-drop",
+  },
+  marketplace: {
+    title: "Marketplace",
```

Comment:
same here, "Create your own Marketplace"? I feel like seeing "Relevant templates" "NFT Drop" is not gonna make people click